src/connection.go

```diff
@@ -62,11 +70,15 @@ func newRedisCon(hostname string, port int, unixSocket string, password string)
 	default:
 		return nil, fmt.Errorf("Redis connection failed, cannot connect either through TCP or Unix Socket")
 	}
-	return redisConn{c}, nil
+
+	return redisConn{
+		c:               c,
+		renamedCommands: make(map[string]string),
```

Comment:
There shouldn't be a need to initialize (`make`) this map, reads from a `nil` map are fine and behave the same way as reads from an empty map.